conf/layer.conf

```diff
@@ -2,8 +2,11 @@
 # to BBPATH as we want to override archiver.bbclass
 BBPATH := "${LAYERDIR}:${BBPATH}"
 
-# We have a recipes directory, add to BBFILES
-BBFILES += "${LAYERDIR}/recipes*/*/*.bb ${LAYERDIR}/recipes*/*/*.bbappend"
+# We have recipes-* and packagegroups directories, add to BBFILES
+BBFILES += "${LAYERDIR}/recipes-*/*/*.bb \
+            ${LAYERDIR}/recipes-*/*/*.bbappend \
+            ${LAYERDIR}/packagegroups/*.bb \
+            ${LAYERDIR}/packagegroups/*.bbappend"
```

Comment:
I'm unable to select multiple lines somehow but I would highly recommend to NOT have a non-standard path for packagegroups.

This makes modifying them in other layers much harder as those layers would have to also modify their BBFILES to add this path.

Just have the packageroups in recipes-nymea/nymea instead?

OE-Core puts some in recipes-core/packagegroups instead, so that's also an option.

Reply:
Never mind, that's not how bitbake works at all, the path only matters within the layer, you can append to it from any other layer at any path, provided it matches a path in BBFILES.

It's still odd, but it's not actually an issue, sorry for the noise.